doc/api/tls.md

```diff
@@ -297,11 +320,11 @@ The following illustrates resuming a TLS session:
 ```js
 const tlsSessionStore = {};
 server.on('newSession', (id, data, cb) => {
-  tlsSessionStore[id.toString('hex')] = data;
+  tlsSessionStore[id.tostring('hex')] = data;
   cb();
 });
 server.on('resumeSession', (id, cb) => {
-  cb(null, tlsSessionStore[id.toString('hex')] || null);
+  cb(null, tlsSessionStore[id.tostring('hex')] || null);
```

Comment:
yes. I was staring and staring at this, but I didn't see the case change. Now I do.